### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by alfredonodo**

### CODE REVIEW

**Review by dtslvr — CHANGES_REQUESTED** [review]
Please resolve the conflicts.

File: apps/client/src/locales/messages.es.xlf [hunk 1]
```diff
@@ -968,15 +968,15 @@
       </trans-unit>
       <trans-unit id="e5c369abfbe1bd70f577aa03b2679797e38d7590" datatype="html">
         <source> Fees for <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </source>
-        <target state="translated">Comisiones por <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transacción} otras {transacciones}}"/> </target>
+        <target state="translated">Comisiones por <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} otras {transactions}}"/> </target>
```
[the remark this answers is not in the document]
Comment (alfredonodo) [reply apps/client/src/locales/messages.es.xlf:971]: I checked the other languages and none translated the word transaction, as it should be a variable name. Or am I wrong?

File: apps/client/src/locales/messages.es.xlf [hunk 2]
```diff
@@ -1460,7 +1460,7 @@
       </trans-unit>
       <trans-unit id="220a4641dcde60d1d86ceec62886b1878f1578d3" datatype="html">
         <source>Update account</source>
-        <target state="translated">Editar cuenta</target>
+        <target state="translated">Mejorar cuenta</target>
```
[the remark this answers is not in the document]
Comment (alfredonodo) [reply apps/client/src/locales/messages.es.xlf:1463]: Ok

File: apps/client/src/locales/messages.es.xlf (hunk 1 above)
[the remark this answers is not in the document]
Comment (dtslvr) [reply apps/client/src/locales/messages.es.xlf:971]: Yes, it needs translation. Please see here: https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/locales/messages.de.xlf#L970

[the remark this answers is not in the document]
Comment (dtslvr) [reply apps/client/src/locales/messages.es.xlf:971]: You can probably see the missing translation here 🙂 
https://ghostfol.io/it/home/summary

File: apps/client/src/locales/messages.it.xlf [hunk 3]
```diff
@@ -968,15 +968,15 @@
       </trans-unit>
       <trans-unit id="e5c369abfbe1bd70f577aa03b2679797e38d7590" datatype="html">
         <source> Fees for <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </source>
-        <target state="translated">Commissioni per <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transaction} other {transactions}}"/> </target>
+        <target state="translated">Commissioni per <x id="INTERPOLATION" equiv-text="{{ summary?.ordersCount }}"/> <x id="ICU" equiv-text="{summary?.ordersCount, plural, =1 {transazione} altre {transazioni}}"/> </target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.it.xlf:971]: `other` is a keyword and cannot be translated 🙂

File: apps/client/src/locales/messages.it.xlf [hunk 4]
```diff
@@ -2658,22 +2658,22 @@
...
-        <target state="new">Excluded from Analysis</target>
+        <target state="new">Escluso dall'analisi</target>
         <context-group purpose="location">
           <context context-type="sourcefile">apps/client/src/app/components/portfolio-summary/portfolio-summary.component.html</context>
           <context context-type="linenumber">176</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="ac598d664f86ba5783915d65f2664a7f38a9d23a" datatype="html">
+        <source>Account Type</source>
+        <target state="new">Tipo di account</target>
```
Comment (dtslvr) [inline apps/client/src/locales/messages.it.xlf:2671]: `state="translated"`

File: apps/client/src/locales/messages.es.xlf (hunk 2 above)
[the remark this answers is not in the document]
Comment (dtslvr) [reply apps/client/src/locales/messages.es.xlf:1463]: There should not be any change in this file

**Review by dtslvr — CHANGES_REQUESTED** [review]

File: apps/client/src/locales/messages.it.xlf [hunk 5]
```diff
@@ -2658,22 +2658,22 @@
...
-        <target state="new">Account Type</target>
-        <context-group purpose="location">
-          <context context-type="sourcefile">apps/client/src/app/components/account-detail-dialog/account-detail-dialog.html</context>
-          <context context-type="linenumber">25</context>
-        </context-group>
-      </trans-unit>
       <trans-unit id="98fc3013bfcbf452b9f37bbfcdb77b9b882866e3" datatype="html">
         <source>Excluded from Analysis</source>
-        <target state="new">Excluded from Analysis</target>
+        <target state="new">Escluso dall'analisi</target>
```
[the remark this answers is not in the document]
Comment (alfredonodo) [reply apps/client/src/locales/messages.it.xlf]: Ok, I followed the German version where there is new https://github.com/ghostfolio/ghostfolio/blob/main/apps/client/src/locales/messages.de.xlf.

File: apps/client/src/locales/messages.it.xlf (hunk 3 above)
Comment (alfredonodo) [reply apps/client/src/locales/messages.it.xlf]: I have already been involved in other translation projects and variables are usually enclosed in {} brackets.

**Review by dtslvr — APPROVED** [review]
Looks fine now! Thank you very much @alfredonodo 👍🏻

**Comment by alfredonodo** [comment]
> Looks fine now! Thank you very much @alfredonodo 👍🏻
You are welcome. Next time I will know what to do.

**Pull request merged**